Week01/Problem04/yuqingwang_04.py

```diff
@@ -0,0 +1,23 @@
+def find():
```

Comment:
I can tell that you're more used to C...while the way you've formatted your code is not against the Python syntax rules, Python code is typically formatted according to something called [PEP8](https://www.python.org/dev/peps/pep-0008/), which (along with all the other PEP's), like the C standard, defines the so-called "best practices" for writing Python code.  Again, nothing you've done is explicitly against Python syntax, but if you have any need to write Python code in a future collaboration with folks who develop mainly in Python, they may *force* you to adhere to PEP8.